### PR TITLE
add more layers

### PR DESCRIPTION
**Pull request by ehesch**
Add layers for planned and search areas: 

Layers = 
1) parks - existing (what was there before)
2) trails - existing (there before)
3) parks - planned (new!)
4) trails - planned (new!)
5) parks - search (new!)
6) trails - search (new!)

Next ideas:
a) add toggle for Agency (maybe using [`L.control.layers.tree`](https://github.com/jjimenezshaw/Leaflet.Control.Layers.Tree) with the "children" as the implementing agencies). 
b) add buffer zone calculations of the various metrics

### CODE REVIEW

File: data-raw/park_trail_geog.R [hunk 1]
```diff
@@ -18,18 +18,45 @@ download.file("ftp://ftp.gisdata.mn.gov/pub/gdrs/data/pub/us_mn_state_metc/plan_
   destfile = temp
 )
 
-parks <- sf::read_sf(unzip(temp, "plan_parks_regional.gpkg")) %>%
-  filter(STATUS == "Existing") %>%
-  group_by(PARKNAME, Label, AGENCY) %>%
+parks_temp <- sf::read_sf(unzip(temp, "plan_parks_regional.gpkg")) %>%
+  # filter(STATUS == "Existing") %>%
+  mutate(AGENCY = recode(AGENCY,
```
Comment (eroten) [inline data-raw/park_trail_geog.R:23]: Just wondering, why did you recode the agency names?

File: data-raw/park_trail_geog.R [hunk 2]
```diff
@@ -18,18 +18,45 @@ download.file("ftp://ftp.gisdata.mn.gov/pub/gdrs/data/pub/us_mn_state_metc/plan_
...
+                         "Carver County Parks and Recreation" = "Carver County",
+                         "Ramsey County Parks and Recreation" = "Ramsey County",
+                         "Dakota County Parks" = "Dakota County",
+                         "Minneapolis Park and Recreation Board" = "Minneapolis",
+                         "Washington County Parks" = "Washington County",
+                         "St Paul Parks And Recreation" = "St. Paul",
+                         "St Paul Parks and Recreation" = "St. Paul",
+                         "St. Paul Parks and Recreation" = "St. Paul",
+                         "Scott County Parks" = "Scott County",
+                         "Scott County/Three Rivers Park District" = "Scott County",#this is the Scott County Regional Trail
```
Comment (eroten) [inline data-raw/park_trail_geog.R:38]: Lets double check with Darcie on which label we should use. I'm also inclined to replace the "/" with "-".

**Review by eroten — CHANGES_REQUESTED** [review]
Ellen,

this PR looks great! I made a few inline comments and would like you to make one change. 

- Add `.DS_Store` to `.gitignore`. This can be done through the R console with `usethis::use_git_ignore(".DS_Store")`.  I'll make a note that this should be added to our repository template in the future. 
  - You may also consider adding `.DS_Store` to your global gitignore,  located in "`~/.gitignore_global`"

I also have some food for thought

- We (CD-Research) are working to standardize things like park and agency names. The tables in Fred use "Three Rivers Park District" and  "Minneapolis Park and Recreation Board", but also use "St. Paul" instead of "St. Paul Parks and Recreation." 🤷‍♀️   We may want to consider which label we use in which context. I'll make a note on this when I review the database tables. 
  - One option is to make two columns,  `Agency_name` and `Agency_name_full`, and use the most appropriate label for the situation.

This is my first PR review, and I'm so happy to have you on the team! 🥰

File: data-raw/park_trail_geog.R (hunk 1 above)
Comment (ehesch) [reply data-raw/park_trail_geog.R:23]: I'll change 'em back (and update to the ones you were using) - there were just (more) inconsistencies in the planned trail/park databases, so was accounting for that.

**Review by eroten — APPROVED** [review]

**Pull request merged**